engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -387,7 +585,10 @@ Please enter one of the following commands when using the cloud-sim extension:
                 -i 
                     - Specify the exact carma-config to use for launching the vehicle and simulation
                       For example " -i image usdotfhwastol/carma-config:carma-system-1.0.1 "
-                      This option is an alternative to -v and should only be used if that option is not specified. 
+                      This option is an alternative to -v and should only be used if that option is not specified.
+            init
```

Comment:
Just for consistency sake, maybe this is better called "init-resources" if the inverse is called "cleanup-resources"?